### ISSUE TITLE
Fix scale/rotate/translate hotkeys broken in the mesh importer UI

### ISSUE THREAD

**Issue by adamkewley** [ComputationalBiomechanicsLab/opensim-creator#1072]
Currently, they don't change the manipulation mode (they used to!)

**Comment by adamkewley** [comment]
This issue was due to the addition of "extended navigation" via ImGui, which was added in 0.5.24 to make keyboard navigation easier, but it had the unintended consequence of making ImGui adsorb many more keyboard events, which bricked the mesh importer.

This is a big enough issue that 0.5.25 should be rushed out before users start getting irritated by the broken keyboard navigation.
